staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go

```diff
@@ -327,22 +322,41 @@ var (
 			},
 		},
 	)
-	SuggestedFlowSchemaWorkloadLeaderElection = newFlowSchema(
-		"workload-leader-election", "leader-election", 200,
+	// We add an explicit rule for endpoints-controller with high precedence
+	// to ensure that those calls won't get catch by the following
+	// <workload-leader-election> flow-schema.
+	//
+	// TODO(#80289): Get rid of this rule once we get rid of support for
+	//   using endpoints and configmaps objects for leader election.
+	SuggestedFlowSchemaEndpointsController = newFlowSchema(
+		"endpoints-controller", "workload-high", 150,
 		flowcontrol.FlowDistinguisherMethodByUserType,
 		flowcontrol.PolicyRulesWithSubjects{
-			Subjects: kubeSystemServiceAccount(flowcontrol.NameAll),
+			Subjects: append(
+				users(user.KubeControllerManager),
+				kubeSystemServiceAccount("endpoints-controller")...),
```

Comment:
The service account name is actually `endpoint-controller`:

https://github.com/kubernetes/kubernetes/blob/770847e6b06d89326f0f2d42c26487cc7f9eae06/plugin/pkg/auth/authorizer/rbac/bootstrappolicy/testdata/controller-role-bindings.yaml#L135-L138

Also include `endpointslicemirroring-controller` which reads from endpoints

Reply:
@liggitt I fixed it in #107054  - `endpoint-controller` and `endpointslicemirroring-controller`

Reply:
done